2_download/src/fetch_wqp_data.R

```diff
@@ -99,8 +99,7 @@ add_download_groups <- function(sitecounts_df, max_sites = 500, max_results = 25
         arrange(desc(results_count)) %>%
         mutate(task_num = MESS::cumsumbinning(x = results_count, 
                                               threshold = max_results, 
-                                              maxgroupsize = max_sites),
-               download_grp = paste0(grid_id,"_",task_num))
```

Comment:
I think this deletion will also need to happen for the `sitecounts_grouped_bad_ids` below!